server/webhookNotifier.ts

```diff
@@ -26,11 +26,21 @@ const REGEX_EMOJI = /([\uE000-\uF8FF]|\uD83C[\uDF00-\uDFFF]|\uD83D[\uDC00-\uDDFF
 const getFilteredContent = (content, address) => {
   let event;
   if (content.chainEvent && content.chainEventType) {
-    event = SubstrateEvents.Label(
-      content.chainEvent.block_number,
-      content.chainEventType.chain,
-      content.chainEvent.event_data
-    );
+    // Check to see if it's an Erc20 event... TODO is there a better way to do this?
```

Comment:
Don't worry too much about this part -- this code is broken on master anyway, as it ignores Moloch/Marlin events. We should ideally expose an ERC20 specific helper function that checks the subscriber to see "is this token subscribed to?" but it doesn't seem necessary for getting this project off the ground.